procdiscovery/pkg/inspectors/nginx/nginx.go

```diff
@@ -11,14 +14,45 @@ import (
 // but in order to avoid huge refactoring we are adding it as a language for now
 type NginxInspector struct{}
 
-const NginxProcessName = "nginx"
+const (
+	NginxProcessName  = "nginx"
+	NginxVersionRegex = `nginx/(\d+\.\d+\.\d+)`
+)
```

Comment:
Should these be private?

```suggestion
const (
	nginxProcessName  = "nginx"
	nginxVersionRegex = `nginx/(\d+\.\d+\.\d+)`
)
```